starter_code/script.js

```diff
@@ -2,8 +2,65 @@ window.onload = function() {
   document.getElementById("start-button").onclick = function() {
     startGame();
   };
+};
+
+  function startGame () {
+    drawGameBoard();
+    drawCar();
+  };
 
-  function startGame() {
+  function drawGameBoard() {
+      var canvas = document.getElementById("canvas");
+      var ctx = canvas.getContext("2d");
+        ctx.fillStyle = "green";
+        ctx.fillRect(30, 30, 400, 1000);
+        ctx.fillStyle = "grey";
+        ctx.fillRect (80,30,300,1000);
+        ctx.fillStyle = "white";
+        ctx.fillRect (85, 10, 10, 1000);
+        ctx.fillStyle = "white";
+        ctx.fillRect (365, 10, 10, 1000);
+        ctx.fillStyle = "white";
+        ctx.fillRect (225, 10, 5, 25);
```

Comment:
To draw this line, you could have used `setLineDash`, you code would be shorter !